packages/calcite-components/src/components/slider/slider.e2e.ts

```diff
@@ -204,6 +204,17 @@ describe("calcite-slider", () => {
     });
   });
 
+  it("step floating point precision", async () => {
+    const page = await newE2EPage();
+    await page.setContent(
+      `<calcite-slider value="1.4" label-handles max="10" min="0.1" snap step="0.1"></calcite-slider>`,
```

Comment:
Should've spotted this sooner, but can you use the `html` util here for consistent formatting?